integration-test/src/test/java/com/sonar/it/java/suite/PmdIT.java

```diff
@@ -48,8 +47,8 @@ static void startSonar() {
     }
 
     @ParameterizedTest
-    @EnumSource(value = JavaVersion.class, mode = EnumSource.Mode.INCLUDE, names = {"JAVA_1_8", "JAVA_11", "JAVA_17", "JAVA_21", "JAVA_22"})
-    void testPmdExtensionsWithDifferentJavaVersions(JavaVersion version) {
+    @EnumSource(value = DefinedJavaVersion.class, mode = EnumSource.Mode.INCLUDE, names = {"JAVA_1_8", "JAVA_11", "JAVA_17", "JAVA_20", "JAVA_21", "JAVA_22", "JAVA_23", "JAVA_24", "JAVA_24_PREVIEW"})
```

Comment:
should/can we remove older non-TLS versions, e.g. `JAVA_20`, `JAVA_22` -> shorten IT duration, do not think it brings much exta

Reply:
Yes, just wanted to make sure they work